numpy/core/src/common/npy_cpu_features.c.src

```diff
@@ -76,12 +76,14 @@ npy__cpu_getxcr0(void)
 #if defined(_MSC_VER) || defined (__INTEL_COMPILER)
     return _xgetbv(0);
 #elif defined(__GNUC__) || defined(__clang__)
+    /* named form of xgetbv not supported on OSX, so must use byte form, see:
+     https://github.com/asmjit/asmjit/issues/78
+   */
     unsigned int eax, edx;
-    __asm__("xgetbv" : "=a" (eax), "=d" (edx) : "c" (0));
-    return (eax | (unsigned long long)edx << 32);
+    __asm(".byte 0x0F, 0x01, 0xd0" : "=a"(eax), "=d"(edx) : "c"(0));
```

Comment:
Why wasn't this tripping CI previously? Is this code never called currently?

Reply:
The tripping CI is succeed again, This code will called if compiled in X86 machine.

Reply:
@matip, by default GCC use macOS assembler (/usr/bin/as) that validating instructions during compile time, so to tripping it we need GCC with an old version of macOS. back in days, I used to replace it with clang assembler to avoid such issues like this. [see this one.](https://gist.github.com/xianyi/2957847)